src/FSAL/FSAL_CORTXFS/handle.c

```diff
@@ -1977,10 +2079,14 @@ static fsal_status_t cfs_file_open(struct kvsfs_file_state *state,
 				   fsal_openflags_t openflags,
 				   struct kvsfs_fsal_obj_handle *obj)
 {
+	fsal_status_t result;
+	perfc_trace_inii(PFT_CFS_OPEN, PEM_CFS_TO_NFS);
```

Comment:
CFS tags should not be used from handle.c file

Reply:
which is convenient function tag to use from handle.c?

Reply:
Pls remove CFS sub string, i.e. PFT_FILE_OPEN